### ISSUE TITLE
Cron job for RBEnv

### ISSUE THREAD

**Issue by vegemite4me** [will-in-wi/letsencrypt-webfaction#20]
Hi,
 Firstly thank you so much for making this possible! It took me a while, probably due to being on web217, but I got there in the end thanks to your readme, the issues, and various wiki pages.

 I was unable to download `cacert.pem` from https://curl.haxx.se/ca/cacert.pem, so ended up getting it from http://mirrors.ircam.fr/pub/crux/core/3.2/ca-certificates/cacert.pem instead. It appeared that the haxx.se website was completely down this morning although it appears to be back up now. Maybe I should have waited, but the ircam.fr file appears to work.

 To get your script working I had to use the following command:

`SSL_CERT_FILE=~/cacert.pem letsencrypt_webfaction --config ~/le_config.yml --domains example.com,www.example.com --public ~/webapps/htdocs/`

 I am now setting up the cron job and wanted to make sure the command you have given in your readme will work.

`0 4 1 */2 *     RBENV_ROOT=~/.rbenv RBENV_VERSION=2.3.0 ~/.rbenv/bin/rbenv exec letsencrypt_webfaction --config ~/le_config.yml --domains example.com,www.example.com --public ~/webapps/htdocs/`

 1. Should I also add the SSL_CERT_FILE environment variable onto the beginning of that crontab command?

 2. And is RBEnv 2.3.0 correct? I seem to have 2.2.4 installed, so should I use the lower version instead?

```
[myuser@web217 ~]$ rbenv version
2.2.4 (set by /home/myuser/.ruby-version)
[myuser@web217 ~]$ rbenv versions
  system
  2.1.0
* 2.2.4 (set by /home/myuser/.ruby-version)
[myuser@web217 ~]$ 
```

Thank you
Robin

**Comment by will-in-wi** [comment]
## TL;DR

1. Yes
2. Use the lower version you have installed.

## Longer answer:

Thanks for the comments, and I'm happy that you are finding this useful!

I'll try and find another source for the `cacert.pem` file to make it easier for others in the future. Thanks for pointing that out! (Tracking in https://github.com/will-in-wi/letsencrypt-webfaction/issues/22)

As you observed, you'll need to add the `SSL_CERT_FILE=~/cacert.pem` line to the beginning of your crontab line to make the script pick it up.

Since you are on a web2xx server, you have CentOS 5, which as you observed makes life substantially more difficult. Kudos on getting it to work! Sadly, without building a lot more of the toolchain and libraries, Ruby 2.3 won't work, which is why I recommended you drop back to 2.2.4. So, yes, use the version you have installed.

I hope this helps! Let me know if you have further issues.
